docs/flink-getting-started.md

```diff
@@ -634,6 +634,26 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+## UPSERT mode
+
+Iceberg supports `UPSERT` based on the primary key when writing data into v2 table format. 
```

Comment:
Here, mention `There are two ways to enable upsert`. the following two parts can be labeled as `1.` and `2.`